bingo.js

```diff
@@ -664,12 +858,40 @@ function createGoalExport()
 		});
 	});
 	$("#export textarea").text(JSON.stringify(result));
-	$("#export").show();
+	openPopup("#export");
+}
+
+function showLinkWithProgress(element)
+{
+	var urlWithProgress = generateNewUrlParamWithProgress(generateSquareProgress());
+	pushNewUrl(urlWithProgress);
+	if (! localStorage.getItem(WAS_SAVE_POPUP_OPEN_KEY)) {
+		$('.js-save-progress-limit').text(SHEET_PROGRESS_SAVE_LIMIT);
+		$('#bingo-box').addClass(SHOW_POPUP_MENU_CLASS_NAME);
+		openPopup("#urlWithProgress");
+		pushNewLocalSetting(WAS_SAVE_POPUP_OPEN_KEY, true);
+	} else if(element !== undefined) {
+		$(element).addClass('success--ok');
+		setTimeout(function() {
```

Comment:
[jQuery's `delay()` function ](https://api.jquery.com/delay/) can be used here.

Reply:
Not really, `addClass`/`removeClass` aren't a part of queue, they grouped and executes as single 'command': https://stackoverflow.com/a/4544152

Reply:
oh :-(